scipy/optimize/zeros.py

```diff
@@ -233,6 +233,21 @@ def bisect(f, a, b, args=(),
         Object containing information about the convergence.  In particular,
         ``r.converged`` is True if the routine converged.
 
+    Examples
+    --------
+    The bisection algorithm fails if `f(a)` and `f(b)` have the same sign.
+
+    >>> def f(x):
+    ...     return (x**2 - 1)
+
+    >>> from scipy import optimize
+
+    >>> root = optimize.bisect(f,0,2)
+    >>> root
+    1.0
+    >>> root = optimize.bisect(f,-2,2)
+    ValueError: f(a) and f(b) must have different signs
```

Comment:
This won't work. Our doctest checkers runs test code to verify it doesn't raise exceptions, so this test will fail. I suggest to just leave this out